shared/teams/confirm-modals/confirm-remove-from-channel.tsx

```diff
@@ -34,28 +34,30 @@ const ConfirmRemoveFromChannel = (props: Props) => {
   const nav = Container.useSafeNavigation()
   const onCancel = React.useCallback(() => dispatch(nav.safeNavigateUpPayload()), [dispatch, nav])
 
-  // TODO(Y2K-1592): do this in one RPC
+  const removeFromChannel = Container.useRPC(RPCChatGen.localRemoveFromConversationLocalRpcPromise)
+
   const onRemove = () => {
-    dispatch(
-      TeamsGen.createChannelSetMemberSelected({
-        clearAll: true,
-        conversationIDKey,
-        selected: false,
-        username: '',
-      })
+    setWaiting(true)
+    setTimeout(() => setWaiting(false), 1000)
+    removeFromChannel(
+      [{convID: ChatTypes.keyToConversationID(conversationIDKey), usernames: members}],
+      _ => {
+        setWaiting(false)
+        dispatch(
+          TeamsGen.createChannelSetMemberSelected({
+            clearAll: true,
+            conversationIDKey,
+            selected: false,
+            username: '',
+          })
+        )
+        dispatch(nav.safeNavigateUpPayload())
+        dispatch(TeamsGen.createLoadTeamChannelList({teamID}))
+      },
+      err => {}
```

Comment:
`teams/confirm-modals/confirm-remove-from-channel.tsx(57,7): error TS6133: 'err' is declared but its value is never read.`

Reply:
oops, that should be hooked up. will do, thanks